kotlinpoet/src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -147,14 +148,26 @@ public class FileSpec private constructor(
 
     val importedTypeNames = codeWriter.importedTypes.values.map { it.canonicalName }
     val importedMemberNames = codeWriter.importedMembers.values.map { it.canonicalName }
+
+    // If we don't have default imports or are collecting them, we don't need to filter
+    var defaultImportsFilter: (String) -> Boolean = { false }
```

Comment:
nit: maybe call it `isDefaultImport`? `filterNot(isDefaultImport)` then becomes slightly easier to read.

Reply:
e2bfd77